vmdb/app/helpers/provider_foreman_helper/textual_summary.rb

```diff
@@ -0,0 +1,37 @@
+module ProviderForemanHelper
+  module TextualSummary
```

Comment:
TBH, this doesn't matter nearly as much as stuff like `x_button` so I'm fine if we leave it the way it is.